src/t8_forest.h

```diff
@@ -342,6 +342,9 @@ void                t8_forest_set_balance (t8_forest_t forest,
                                            const t8_forest_t set_from,
                                            int no_repartition);
 
+/* NOTE write description */
+void                t8_forest_set_subelements (t8_forest_t forest, const t8_forest_t set_from);
```

Comment:
If we would use an additional flag parameter for the set_subelements function, as suggested in the comment above, the name might be more reasonable since the function could be used eighter to remove hanging nodes (as a special case) or for some other reason and therefore for a broader range of use cases.

Reply:
Same as above. I will focus on the remove hanging faces case for now and maybe add a more general infrastructure for other use cases in the future.